runtime/dolphin/src/lib.rs

```diff
@@ -1069,9 +1161,10 @@ construct_runtime!(
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 41,
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 42,
 
-		// Assets management
+		// Asset and Private Payment
 		Assets: pallet_assets::{Pallet, Storage, Event<T>} = 45,
```

Comment:
BTW if you want to see the `Assets` tab in the front end you'll need to add `Call` here, otherwise it won't load.
I accidentally removed it after finishing testing...
Correcting it in my pr as well. https://github.com/Manta-Network/Manta/pull/447

Reply:
Not sure whether we need that actually. Adding that for now (seems no harm at least).